README.md

```diff
@@ -71,6 +71,7 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade 15 (Early 2020) Base              |  1532:0255  |
 | Razer Blade Pro 17 (Early 2020)               |  1532:0256  |
 | Razer Huntsman Mini                           |  1532:0257  |
+| Razer Huntsman Mini (JP)                      |  1532:0269  |
```

Comment:
Please keep this list sorted by usb pid, so put this after Razer Huntsman V2 Analog